elements/bulbs-video/elements/meta.js

```diff
@@ -0,0 +1,66 @@
+import React, { PropTypes } from 'react';
```

Comment:
I was pushing a hard line on  creating a `Root` component for elements that separates store wiring stuff from top-level rendering logic.

Now that I'm going through and implementing more elements I'm not so attached to that and can't make a cogent argument for it.